pyglet/lib.py

```diff
@@ -250,6 +250,16 @@ def find_library(self, path):
     @staticmethod
     def load_framework(name):
         path = ctypes.util.find_library(name)
+
+        # Hack for compatibility with macOS > 11.0
+        if path is None and sys.platform == 'darwin':
```

Comment:
Since this method is part of the `MachOLibraryLoader` class, I don't think we need the `sys.platform` check here.

Reply:
Yeah I wasn't sure about that one, but you're right, it could definitely be removed from both this file and the other file in this pull request